src/main/java/org/jenkinsci/test/acceptance/po/Parameter.java

```diff
@@ -53,6 +53,11 @@ public By path(String rel) {
             return super.path(rel);
         }
 
+        if (Boolean.getBoolean("new-job-page.flag.defaultValue")) {
+            return by.xpath(
+                    "//input[@name='name' and @value='%s']/parent::div[@name='parameter']//*[@name='%s']", name, rel);
```

Comment:
time to add `data-testid=parameter-${name}` so things can be found.

However the location structure is identical between the existing and experimental page?dialog vs standard page. 
The only difference is that we are inside a dialog. In other words this should be the same code for both cases given the search is in the whole document `//...`